src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -15,13 +15,34 @@
 
 namespace Microsoft.SqlTools.ServiceLayer.EditData.UpdateManagement
 {
+    /// <summary>
+    /// An error indicating that a delete action will delete multiple rows.
+    /// </summary>
+    public class DeleteError : Exception
```

Comment:
Could you rename this class to better follow .Net naming conventions for exception classes?  For example, maybe something like `EditDataDeleteException` or something like that.